operatorconfig/moduleconfig/common/version-values.yaml

```diff
@@ -24,3 +24,7 @@ powerflex:
   v2.6.0:
       authorization: "v1.6.0"
       observability: "v1.5.0"
+  v2.7.0:
+      authorization: "v1.6.0"
+      observability: "v1.5.0"
```

Comment:
Hi @KshitijaKakde Can you please pull the latest changes from main branch in your private branch, I believe there will be some conflicts in this file, since one more new module have been added for this version and rest module's version need to be updated. Version in your current PR for rest modules seems to be the latest one, that is correct.